### DISCUSSION TITLE
Welcome to Discussions of Sentinel!

### DISCUSSION THREAD

**Discussion by sczyh30** [alibaba/Sentinel#1888] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by llly0031** [comment]
sentinel读不到微服务资源， 有的服务能读到，有的读不到
